release-please-config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "bootstrap-sha": "ac251ea3b034166f94172f932e10f5920b3aec42",
+  "sequential-calls": true,
+  "packages": {
+    ".": {
```

Comment:
One question here would be that do we want to use the default `changelog-sections` option (see the default config at https://git.io/JqCZL) that hides, for example, `chore`, `refactor` and `docs` type of commits or would we rather want to show some or all of the hidden ones too?

I think `docs` would be good to be shown when the documentation has been updated (as we are also planning to bring the API Swagger documentation to be part of this repo). `chore` sometimes includes pretty noticable version updates like the Django upgrade.

Reply:
I created [OH-73](https://helsinkisolutionoffice.atlassian.net/browse/OH-73)
I think `changelog-sections` configuration should do in another PR